content/en/user-manual/api/project-archive.md

```diff
@@ -19,9 +19,15 @@ The request will start an archive job and the job details will be returned in th
 ## Example
 
 ```none
-curl -H "Authorization: Bearer fdslkjlk32j2l3kj2lkj2lkj323rr" -H "Content-Type: application/json" -X POST https://playcanvas.com/api/projects/99999999/export
+curl -H "Authorization: Bearer fdslkjlk32j2l3kj2lkj2lkj323rr" -H "Content-Type: application/json" -X POST -d '{"branch_id": "4c308c6d-e213-4ca1-afbb-6d87ff8edd9a"}' https://playcanvas.com/api/projects/99999999/export
```

Comment:
I hope that's not a real branch id?

Reply:
Might be better to use all 99's like the project id after it

Reply:
Changed to 99s